src/data/codingcontracttypes.ts

```diff
@@ -307,15 +307,7 @@ export const codingContractTypesMetadata: ICodingContractTypeMetadata[] = [
         reach = Math.max(i + data[i], reach);
       }
       const solution: boolean = i === n;
-
-      if (ans === "1" && solution) {
-        return true;
-      }
-      if (ans === "0" && !solution) {
-        return true;
-      }
-
-      return false;
+      return ans === "1" && solution || ans === "0" && !solution;
```

Comment:
shouldn't we add parenthesis for better readability

Reply:
You convinced me me with JS doing weird stuff